Puzzlepart/prosjektportalen365
# fix for #648 using /_api/siteiconmanager/getsitelogo instead of base64 from graph

Fix for issue #648 loading site logos the same way as they are loaded within the teamsites themselves in sharepoint

### Your checklist for this pull request


### Description

Removed all base64 based site logo logic and pointing image src directly to `/_api/siteiconmanager/getsitelogo`. 

Some initial testing results here 

before:
https://user-images.githubusercontent.com/170147/184892332-33c51112-8adf-4f15-99f9-e5d038e77f06.mp4

After 
https://user-images.githubusercontent.com/170147/184892402-92129014-9fc6-402d-a8b6-32abb4f8d22b.mp4



### How to test

if running locally, 
1. watch with `npm run watch`
2. Navigate to e.g.

https://prosjektportalen.sharepoint.com/sites/pp365/SitePages/Home.aspx?debug=true&noredir=true&debugManifestsFile=https://localhost:4321/temp/manifests.js

If testing this with a deployed build, just navigate to the front page of the portfolio, e.g. https://prosjektportalen.sharepoint.com/sites/pp365 and observe all photos loading at the same time.

### Update of documentation
- [x] Update user manual if needed

### Relevant issues (if applicable)

💔Thank you!
